deploy/lambda/whitelist.py

```diff
@@ -0,0 +1,80 @@
+import json
+import boto3
+import botocore
+from botocore.vendored import requests
+
+# Config global var before deployment
+REMOVE_PORTS = []
+WHITELIST_SERVICE = 'https://my.service.org/whitelist'
+
+
+def get_nodes_ips():
+    data = requests.get(url=WHITELIST_SERVICE)
+    return json.loads(data.json()['body'])
+
+
+def get_ip_perm_module(ip):
+    return {
+        'IpProtocol': "tcp",
+        'FromPort': 11625,
+        'ToPort': 11625,
+        'IpRanges': [{'CidrIp': ip, 'Description': f'Node - {ip}'}]
```

Comment:
i don't know if this is for allowing a specific node to access this one, but if so you can assign that node's security group id instead of it's ip. it's more generic and doesn't depent on ip changes